introduction_to_amazon_algorithms/jumpstart-foundation-models/question_answering_retrieval_augmented_generation/question_answering_Cohere+langchain_jumpstart.ipynb

```diff
@@ -0,0 +1,1702 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    Cohere_Generate_Endpoint_Name = "Endpoint-Cohere-Generate-Model-Command-Light-01"
</pre><p>could you please use prefix of jumpstart-example-raglc- as the endpoint name?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4409/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
The prefix was added in the endpoint name.